android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -57,6 +57,8 @@ data class QuestionnaireConfig(
   val generateCarePlanWithWorkflowApi: Boolean = false,
   val cqlInputResources: List<String>? = emptyList(),
   val showClearAll: Boolean = false,
+  val showQuestionnaireRequiredAsterisk: Boolean = true,
+  val showQuestionnaireRequiredText: Boolean = false,
```

Comment:
If `showQuestionnaireRequiredAsterisk` is set to false does it mean `showQuestionnaireRequiredText` will be used instead? These two properties are they used independently or the absence of one indicates the other should be used?

Reply:
If they are not used independently we can just use one and apply negation where the opposite is desired.

Reply:
I'd suggest we rename them to: 

```
showRequiredTextAsterisk
showRequiredText
```